GothicLlamasAPI/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# DB Credentials
+db.properties
+config.yml
+
+# Intellij related
+.DS_Store
+.idea
+target/
+GothicLlamasAPI/dependency-reduced-pom.xml
+GothicLlamasAPI/target/GothicLlamas-1.0-SNAPSHOT-sources.jar
+GothicLlamasAPI/target/GothicLlamas-1.0-SNAPSHOT.jar
+GothicLlamasAPI/target/maven-archiver/pom.properties
+GothicLlamasAPI/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
+GothicLlamasAPI/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/createdFiles.lst
+GothicLlamasAPI/target/original-GothicLlamas-1.0-SNAPSHOT.jar
+GothicLlamasAPI/target/surefire-reports/is2Test.txt
+GothicLlamasAPI/target/surefire-reports/TEST-is2Test.xml
+GothicLlamasAPI/target/test-classes/is2Test.class
+GothicLlamasAPI/target/maven-status/maven-compiler-plugin/testCompile/default-testCompile/inputFiles.lst
+GothicLlamasAPI/target/GothicLlamas-1.0-SNAPSHOT-sources.jar
+GothicLlamasAPI/target/GothicLlamas-1.0-SNAPSHOT.jar
+GothicLlamasAPI/target/maven-archiver/pom.properties
+GothicLlamasAPI/target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst
+GothicLlamasAPI/target/original-GothicLlamas-1.0-SNAPSHOT.jar
```

Comment:
Everything under `target/` should be ignored - you don't want to have to keep updating this list with new generated files every time you expand your production codebase.